packages/core/test/Cardano/util/primitives.test.ts

```diff
@@ -85,6 +86,30 @@ describe('Cardano.util/primitives', () => {
       expect(() => HexBlob(' 1234567890abcdef')).toThrowError(InvalidStringError);
       expect(() => HexBlob('1234567890abcdefg')).toThrowError(InvalidStringError);
     });
+
+    it('fromBytes converts byte array into HexBlob', () => {
+      expect(HexBlob.fromBytes(new Uint8Array([112]))).toEqual('70');
+    });
+  });
+
+  describe('Base64Blob', () => {
+    it('allows an empty string', () => {
+      expect(() => Base64Blob('')).not.toThrow();
+    });
+
+    it('does not throw when asserting a valid base64 encoded string', () => {
+      // from ogmios.wsp.json
```

Comment:
What's from `ogmios.wsp.json`?

Reply:
The example base64 encoded string. Not a very useful comment in this context, should I remove it?

Reply:
Yeah

Reply:
Amended the commit (didn't do fixup since it's only 1 commit and only 1 change request)